### ISSUE TITLE
docker check - refresh daemon check

### ISSUE THREAD

**Issue by alexschroeter** [jhnnsrs/konstruktor#4]
When Konstruktor checks for a docker daemon, it does not recheck if you press the "Prev" button and go "Next". It should update if you start the daemon in the meantime (a refresh button would also be nice).

I didn't check if it would recognize the daemon to do the GPU check in the next step because I restarted Konstruktor.

**Comment by jhnnsrs** [comment]
Agreed, i am currently in the midst of revamping the forms to more closely fit a "plugin" like interface, so that also
other open-source tools can hook into this (e.g. for deploying OMERO or Jupyterhub). That will be integrated.
